views/404.ejs

```diff
@@ -1,6 +1,8 @@
 <%- include('includes/head.ejs') %>
 </head>
+
 <body>
     <%- include('includes/navigation.ejs') %>
-    <main><h1><%= pageTitle %></h1></main>    
+    <h1>Page Not Found!</h1>
```

Comment:
Opa, lembra que a gente tinha colocado o título dentro do `main` pra ficar com espaço correto na página de page not found?